OOP1 Homework/src/com/oophomework/Person.java

```diff
@@ -1,6 +1,31 @@
 package com.oophomework;
 
 public class Person {
+    private String firstName;
+    private String lastName;
+    private int age;
 
+    //    private com.oophomework.Person[] persoane = new com.oophomework.Person[4];
```

Comment:
Don't leave commented out code in your main branches.
If you need to keep some alternate version of the code for later use, have a special branch/stash set up for that.